src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -115,4 +143,14 @@
         return commands;
     }
 
+    /**
+     * Proxy usage command creation.
+     * @param host Host.
+     * @param port Port.
+     * @return Command string.
+     * @see #proxy
+     */
+    private static String proxyString(final String host, final int port) {
```

Comment:
why so many lines of code if we can just do `String.format()` right in the constructor? this method looks like over-engineering to me.